src/components/modals/SharingModal.js

```diff
@@ -42,6 +46,11 @@ const SharingModal = ({
   isVisible,
   onClose
 }) => {
+
+  const [showQRCode, setShowQRCode] = useState(false);
+
+  const inviteLink = getWindowLocationHref();
```

Comment:
@jimmcgaw To avoid race conditions please run `getWindowLocationHref()` as late as possible. On mobile, the app loads with `window.location.href === 'http://www.leapchat.org'`, with no URL hash :frowning_face: .

Reply:
https://github.com/cryptag/leapchat/pull/331/files#r1125968319